roles/tempo/tasks/main.yml

```diff
@@ -0,0 +1,74 @@
+---
+- name: Create tempo user
+  user:
+    name: tempo
+    create_home: false
+
+- name: Create conf directory
+  file:
+    state: directory
+    path: /etc/tempo
+    mode: '0755'
+    group: tempo
+    owner: tempo
+
+- name: Create data directory
+  file:
+    state: directory
+    path: "{{ tempo_data_path }}"
+    mode: '0755'
+    group: tempo
+    owner: tempo
+
+- name: Download binary
+  get_url: 
+    url: https://github.com/grafana/tempo/releases/download/{{ tempo_version }}/tempo_{{ tempo_version[1:] }}_linux_amd64.tar.gz
+    dest: /tmp/tempo_linux_amd64.tar.gz
+    checksum: sha256:{{ tempo_download_sha256 }}
+  register: tempo_download
+
+- name: Unarchive
+  unarchive:
+    src: /tmp/tempo_linux_amd64.tar.gz
+    dest: /usr/local/bin
+    remote_src: yes
+    owner: tempo
+    group: tempo
+  when: tempo_download.changed
+  notify: restart tempo
+
+- name: binary file permission
+  file:
+    path: /usr/local/bin/tempo
+    owner: tempo
+    group: tempo
```

Comment:
The binaries should be owned by `root:root`, otherwise a security issue in tempo would allow it to modify its own code!

Reply:
https://github.com/ipfs-search/ipfs-search-deployment/commit/5b682ab62418c1aeb72fabb5002035b36527db1b